### PR TITLE
Fix failing docker build

### PR DESCRIPTION
**Pull request by jon-shipley**
Blind fix for failing build on azure

### CODE REVIEW

File: pupil-spa/Dockerfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ WORKDIR /ng-app
 
 COPY . .
 
+RUN yarn run postinstall
```
Comment (GuyHarwood) [inline pupil-spa/Dockerfile]: £1 says this isn't needed :)

Comment (jon-shipley) [reply pupil-spa/Dockerfile]: You owe me £1 😆

Comment (GuyHarwood) [reply pupil-spa/Dockerfile]: Goddamit!

Question is - why is post install not firing automatically?

Comment (jon-shipley) [reply pupil-spa/Dockerfile]: I reckon it does but it then copies as bunch of files into another directory `ng-app`.  We could clean that up by only running install once, in the new directory.

**Pull request merged**